app/src/main/java/com/rob729/quiethours/Adapter/ProfileListAdapter.kt

```diff
@@ -136,4 +136,14 @@ class ProfileListAdapter(
             }
             .show()
     }
+    fun deleteAll() {
+        val size: Int = itemCount
```

Comment:
There was a reason to shift the function to the profileListAdapter. 
Just copy-pasting the code from the previous location was intended.
Use the profiles ArrayList to implement the same in a cleaner way.

Reply:
@rob729 I have now implemented deleteAll() using profiles Arraylist.